sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -816,8 +818,8 @@ public async Task ChecksForCancelation()
             using var stream = new MemoryStream(data);
             await blob.UploadAsync(stream);
 
-            // Create a CancellationToken that times out after .1s
-            CancellationTokenSource source = new CancellationTokenSource(TimeSpan.FromSeconds(.1));
+            // Create a CancellationToken that times out after .01s
+            CancellationTokenSource source = new CancellationTokenSource(TimeSpan.FromSeconds(.01));
```

Comment:
Was this uploading that fast in the same data center or something?

Reply:
Yep, apparently